HeterogeneousCore/SonicTriton/scripts/cmsTriton

```diff
@@ -604,7 +623,14 @@ elif [ "$OP" == start ]; then
 
 	START_EXIT=0
 	for ((counter=0; counter < ${RETRIES}; counter++)); do
-		if [ "$START_EXIT" -ne 0 ]; then make_tmp; fi
+		if [ "$START_EXIT" -ne 0 ]; then
```

Comment:
i was hoping that we could reduce duplication by eliminating these lines from outside the loop and then running them every time here (i.e. without the `"$START_EXIT" -ne 0` condition)

Reply:
Removed duplication.  The list_models function adds onto the REPOS array (which can be initialized based on one of the command line options, so I didn't want to assign it to ()), which is why I only do list_models for the first pass and subsequent ones if we're planning on editing the model configs